src/vue/fields/SelectField.vue

```diff
@@ -84,38 +96,34 @@ export default {
       type: Boolean,
       default: false,
     },
-    formFree: {
-      type: Boolean,
-      required: false,
-      default: false,
-    },
     errorMessage: {
       type: String,
-      default: undefined,
+      default: '',
+    },
+    keyAsValueText: {
+      type: String,
+      default: '',
     },
   },
 
   data: () => ({
-    currentValue: null, // selected item in the list
-    selectedValue: null, // active list element (for arrow navigation)
-    isExpanded: false,
+    selected: null, // selected item in the list
+    preSelected: null, // active list element (for arrow navigation)
+    isListOpened: false,
     KEY_CODES,
   }),
 
   watch: {
-    isExpanded (value) {
+    isListOpened (value) {
```

Comment:
I'd rather place this stuff explicitly in the `openList`

Reply:
moved